# Exercise 2

Count number of alignments that match perfectly to the genome

In [5]:
f = open('/Users/cmdb/data/results/SRR072893.sam')

#define variables
i = 0 #itteratiion count
m = 0

for line in f:
    if line.find('NM:i:0') > -1:
        m += 1
    i += 1

print ('number of matches:', i)
print ('number of exact matches:', m)

f.close()

number of matches: 2000000
number of exact matches: 1702765


Output:

number of matches: 2000000 #Double check to make sure alignments are the same as last time

number of exact matches: 1702765